### DISCUSSION TITLE
Proposal about color used in imports/exports in vscode

### DISCUSSION THREAD

**Discussion by JoseMurilloc** [daltonmenezes/aura-theme#100] [General]
First of all I want to see that it is my favorite theme, just below there is an image of how I have been using it in my environment.
In this case, the exact checks i use:
```
 {
  "scope": [
    "keyword.operator.comparison",
    "keyword.control.loop.js",
    "keyword.control.loop.ts",
    "keyword.control.import.js",
    "keyword.control.import.ts",
    "keyword.control.import.tsx",
    "keyword.control.from.js",
    "keyword.control.from.ts",
    "keyword.control.from.tsx",
    "keyword.control.export",
    "keyword.operator.instanceof.js",
    "keyword.operator.expression.instanceof.ts",
    "keyword.operator.expression.instanceof.tsx"
  ],
  "settings": {
    "fontStyle": "italic bold",
    "foreground": "#e06ae0"
  }
},
```

## Preview
> :warning: *My suggestion is just the color pink*
![vscode](https://user-images.githubusercontent.com/43470555/152405996-f2369ae7-c232-457f-b4a0-93bca3bfd21f.png)

**Comment by daltonmenezes** [comment]
Thanks @JoseMurilloc , but this looks more like a Dracula approach than an Aura one. For now, I think this customization is better left to the user if he/she/they prefers this direction. 💜
